presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -53,8 +53,9 @@ std::string SystemConfig::prestoVersion() const {
   return requiredProperty(std::string(kPrestoVersion));
 }
 
-std::string SystemConfig::discoveryUri() const {
-  return requiredProperty(std::string(kDiscoveryUri));
```

Comment:
If this wasn't specified for spark and startup didn't fail, then what does "requiredProperty" actually mean?

Reply:
I believe this was specified, but the value was wrong, e.g. actual announcement was failing.